cmd/util/root.go

```diff
@@ -0,0 +1,32 @@
+package util
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/consul/api"
+)
+
+func PrintConsulServices() {
+	s, _ := ListConsulServices(api.DefaultConfig())
```

Comment:
please use RuntimeConfig.Options.ConsulOptions.ToConsulConfig() instead of api.DefaultConfig()

Reply:
that makes more sense - I'll pass the `var rc = runner.RunConfig` from `cmd/main.go`
- let me know if you had something else in mind